### PR TITLE
Create GithubWorkflow.RST

### PR DESCRIPTION
**Pull request by yangchen1**
Created doc for Github Workflow.

### CODE REVIEW

File: docs/GithubWorkflow.RST [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#Github Workflow
```
Comment (yuya-oc) [inline docs/GithubWorkflow.RST]: Should be `# GitHub Workflow`.

File: docs/GithubWorkflow.RST [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+When community members step into our repository, it should be:
+- Easy to find issues through proper labels,
```
Comment (Razzeee) [inline docs/GithubWorkflow.RST]: As this is a list, you should get rid of the `,`

File: docs/GithubWorkflow.RST [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Easy to understand what the status of their PR is once submitted, and
```
Comment (Razzeee) [inline docs/GithubWorkflow.RST]: Leave out the `, and`

**Comment by jasonblais** [comment]
Thanks @yangchen1,

I'm going to close this due to inactivity and re-submit with other proposed changes from @yuya-oc, @Razzeee and @lfbrock.

**Pull request closed without merging**